commands/core/cli.drush.inc

```diff
@@ -90,6 +94,64 @@ function _drush_core_cli_get_commands() {
 }
 
 /**
+ * Returns a mapped array of casters for use in the shell.
+ *
+ * These are Symfony VarDumper casters.
+ * See http://symfony.com/doc/current/components/var_dumper/advanced.html#casters
+ * for more information.
+ *
+ * @return array.
+ *   An array of caster callbacks keyed by class or interface.
+ */
+function _drush_core_cli_get_casters() {
```

Comment:
OR maybe the `Drupal\Core\Field\FieldItemInterface` caster could just use `Drupal\Core\TypedData\TypedDataInterface` instead?